adding_species/contributing.ipynb

```diff
@@ -0,0 +1,594 @@
+    "\n",
+    "Tests at the level of individual species are meant to be idiosyncratic, so our example need not be laborious.\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "comic-reconstruction",
+   "metadata": {},
+   "source": [
+    "### Initiating a Pull Request for your species\n",
+    "Okay if we are happy enough with our species we are ready to commit all our changes to our branch and start a pull request on github. First we double check we are on our new branch\n",
+    "```\n",
+    "git branch -v\n",
+    "```\n",
+    "and if all good we are ready to commit with\n",
+    "```\n",
+    "git commit -am \"added mosquito genome woot\"\n",
+    "```\n",
+    "\n",
+    "\n",
+    "\n",
+    "\n",
+    "Part of starting the pull request will be initializing a new issue using the Species QC template seen below\n",
+    "![Figure 2](gh2.png)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "portuguese-designation",
+   "metadata": {},
+   "source": [
+    "# add a demographic history\n",
+    "As part of the Ag1000G paper we estimated population size histories from each population \n",
+    "using the stairwayplot method. I'm going to download one of these estimated histories here and manipulate the file a bit to pull out the times and population sizes going back in time.\n",
+    "\n",
+    "First download the history."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 18,
+   "id": "loaded-yahoo",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "--2021-03-10 16:05:57--  ftp://ngs.sanger.ac.uk/production/ag1000g/phase1/AR3.1/stairway_plot/GAS.meru_mela.sfs.sp.summary\n",
+      "           => ‘GAS.meru_mela.sfs.sp.summary’\n",
+      "Resolving ngs.sanger.ac.uk (ngs.sanger.ac.uk)... 193.62.203.79\n",
+      "Connecting to ngs.sanger.ac.uk (ngs.sanger.ac.uk)|193.62.203.79|:21... connected.\n",
+      "Logging in as anonymous ... Logged in!\n",
+      "==> SYST ... done.    ==> PWD ... done.\n",
+      "==> TYPE I ... done.  ==> CWD (1) /production/ag1000g/phase1/AR3.1/stairway_plot ... done.\n",
+      "==> SIZE GAS.meru_mela.sfs.sp.summary ... 47711\n",
+      "==> PASV ... done.    ==> RETR GAS.meru_mela.sfs.sp.summary ... done.\n",
+      "Length: 47711 (47K) (unauthoritative)\n",
+      "\n",
+      "GAS.meru_mela.sfs.s 100%[===================>]  46.59K  93.2KB/s    in 0.5s    \n",
+      "\n",
+      "2021-03-10 16:06:03 (93.2 KB/s) - ‘GAS.meru_mela.sfs.sp.summary’ saved [47711]\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "!wget ftp://ngs.sanger.ac.uk/production/ag1000g/phase1/AR3.1/stairway_plot/GAS.meru_mela.sfs.sp.summary "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "growing-rally",
+   "metadata": {},
+   "source": [
+    "then i'm going to chop out what i want using a little python. this code is a bit ugly\n",
```

Comment:
I'm of two minds here - unless you're going to go through this bit of parsing code step by step, it might distract some people (most attendees marked their Python skill as "intermediate," so can probably read it but maybe not quickly), so is it really necessary; on the other hand, it's useful to see the specific things you're pulling out of the stairwayplot result. Maybe just highlight the parameters you're pulling out?

Reply:
yeah let me clean this up with a little bit of preprocessing